pipelines/jobs/configurations/jdk16u_pipeline_config.groovy

```diff
@@ -24,7 +15,7 @@ class Config16 {
                 additionalNodeLabels : 'ci.project.openj9 && hw.arch.x86 && sw.os.linux',
                 dockerImage: [
                         hotspot     : 'adoptopenjdk/centos6_build_image',
-                        openj9      : 'adoptopenjdk/centos7_build_image'
+                        openj9      : 'adoptopenjdk/centos7_build_image@sha256:de97e2dd5655e73ddd47f32b3676dae0865aaeb735770b1b5e7fdf0c9bf92fef'
```

Comment:
I don't think this is needed. It can be the latest. Cent6 has an openssl issue which requires an old version of the container but cent7 container is fine afaik. We probably had locked in the version for the release branch.